### PR TITLE
Feature/refactor character selection

### PR DESCRIPTION
**Pull request by uly3**
***ADVICE NEEDED***
Ignore commit 8/30

-Created a class called CharacterSelection.java
-This class holds 4 methods that return the name,sex,age,character
-The name,sex,age methods are mentioned in the picker method to be called and then set the character object's name,age,sex
-picker method allows user to pick race, then turns the dummy character object to that specific child type.
~Problem 1: Can't return character
~Problem 2: In App.java, I probably won't be able to call upon the instantiated class CharacterSelection as followed: 

************************

CharacterSelection pick;
pick.picker();

### CODE REVIEW

**Comment by wtvamp** [comment]
Do you have a diagram to share?

**Comment by uly3** [comment]
One Sec.

**Comment by uly3** [comment]
I uploaded/committed to this branch a new diagram called NewRPG.draw.io

File: src/main/java/com/senecafoundation/CharacterSelection.java [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+        while(flag)
+        {
+
+            System.out.println("\nPick A Character: \n Orc, Viking, DemiHuman, Elf, Nuet, Halfling, HalfElf, ShadowElf, Human, Tiefler");
+            String race = input.nextLine();
+
+            if((race.equals("Orc")) || (race.equals("orc")))
+            {
+                character = new Orc("",0,"",25,20,dataHandler);
+                character.setName(pickName());
```
Comment (wtvamp) [inline src/main/java/com/senecafoundation/CharacterSelection.java]: Can we do something better with these five lines?  So they aren't copy and paste code?  That's usually a sign of stinky code.

**Review by wtvamp — COMMENTED** [review]
I think there's one more change you can make when calling character.setName, setSex, setAge, flag=false, return character - you should only have to do that once.

**Comment by uly3** [comment]
Can I wrap those 5 things into a void method within the picker method then
call that void method for each conditional statement? Is that possible?

**Comment by uly3** [comment]
I just realized that I can’t do that because when using a void method, you
can’t use the keyword “return”

**Pull request merged**